java/main/com/google/privacy/differentialprivacy/LaplaceNoise.java

```diff
@@ -149,9 +152,14 @@ public ConfidenceInterval computeConfidenceInterval(
       long lInfSensitivity,
       double epsilon,
       @Nullable Double delta,
-      double confidenceLevel) {
-    // TODO: Implement confidence interval computation.
-    throw new UnsupportedOperationException("Not implemented yet.");
+      double alpha) {
+    checkConfidenceIntervalParameters(l0Sensitivity, lInfSensitivity, epsilon, delta, alpha);
+    ConfidenceInterval confInt =
+        computeConfidenceInterval(
+            0, Noise.getL1Sensitivity(l0Sensitivity, lInfSensitivity) / epsilon, alpha);
```

Comment:
I would add variable lambda here, would be easier to read.

Also I would rename it to either  interval[AroundZero] or confidenceInterval[AroundZero]. WDYT?